### PR TITLE
feat: add dai to internal testnet hard money markets

### PR DESCRIPTION
**Pull request by karzak**
Updates internal testnet genesis to include dai money market and associated pricefeed changes. Updates seed script to include dai as as evm conversion pair. This enables testing of 18 decimal money markets on internal testnet.

## Checklist
 - [ ] Changelog has been updated as necessary.

### CODE REVIEW

File: ci/env/kava-internal-testnet/genesis.json [hunk 1]
```diff
@@ -3307,10 +3325,28 @@
...
+            "oracles": [
+              "kava1hdn83q3srldcpan4ex2v4npuae8hmnfrps3e6h"
+            ],
+            "active": true
+          },
+          {
+            "market_id": "dai:usd:30",
+            "base_asset": "dai",
+            "quote_asset": "usd",
+            "oracles": [
```
Comment (boodyvo) [inline ci/env/kava-internal-testnet/genesis.json:3341]: Do we have oracle for internal testnet? Or is it just a required value?

File: ci/env/kava-internal-testnet/genesis.json [hunk 2]
```diff
@@ -2344,6 +2344,24 @@
...
           },
+          {
+            "denom": "erc20/eth/dai",
+            "borrow_limit": {
+              "has_max_limit": false,
+              "maximum_limit": "0.000000000000000000",
+              "loan_to_value": "0.500000000000000000"
+            },
+            "spot_market_id": "dai:usd:30",
+            "conversion_factor": "1000000000000000000",
```
Comment (nddeluca) [inline ci/env/kava-internal-testnet/genesis.json:2355]: 10^18 👍

File: .github/scripts/seed-internal-testnet.sh [hunk 3]
```diff
@@ -121,6 +121,7 @@ npx hardhat --network "${ERC20_DEPLOYER_NETWORK_NAME}" mint-erc20 "$AXL_BUSD_CON
 npx hardhat --network "${ERC20_DEPLOYER_NETWORK_NAME}" mint-erc20 "$AXL_XRPB_CONTRACT_ADDRESS" "$DEV_TEST_WALLET_ADDRESS" 1000000000000000000000
 npx hardhat --network "${ERC20_DEPLOYER_NETWORK_NAME}" mint-erc20 "$AXL_BTCB_CONTRACT_ADDRESS" "$DEV_TEST_WALLET_ADDRESS" 1000000000000000000000
 npx hardhat --network "${ERC20_DEPLOYER_NETWORK_NAME}" mint-erc20 "$WBTC_CONTRACT_ADDRESS" "$DEV_TEST_WALLET_ADDRESS" 10000000000000
+npx hardhat --network "${ERC20_DEPLOYER_NETWORK_NAME}" mint-erc20 "$DAI_CONTRACT_ADDRESS" "$DEV_TEST_WALLET_ADDRESS" 10000000000000000000000000
```
Comment (nddeluca) [inline .github/scripts/seed-internal-testnet.sh:124]: 💰

**Review by nddeluca — APPROVED** [review]
I didn't test locally, but I don't see any obvious issues

File: ci/env/kava-internal-testnet/genesis.json (hunk 1 above)
Comment (nddeluca) [reply ci/env/kava-internal-testnet/genesis.json:3341]: Good question, I'm not sure where the private key for `kava1hdn83q3srldcpan4ex2v4npuae8hmnfrps3e6h` is defined 🤔

Comment (karzak) [reply ci/env/kava-internal-testnet/genesis.json:3341]: We just hard code the prices in the pricefeed and set them to never expire: https://github.com/Kava-Labs/kava/blob/ee4fc56f013c06a093f3b5a1202652d7e1e81124/ci/env/kava-internal-testnet/genesis.json#L3555

**Pull request merged**